querier.go

```diff
@@ -266,6 +267,62 @@ func (q *blockQuerier) Close() error {
 	return merr.Err()
 }
 
+// Bitmap used by func isRegexMetaCharacter to check whether a character needs to be escaped.
+var regexMetaCharacterBytes [16]byte
+
+// isRegexMetaCharacter reports whether byte b needs to be escaped.
+func isRegexMetaCharacter(b byte) bool {
+	return b < utf8.RuneSelf && regexMetaCharacterBytes[b%16]&(1<<(b/16)) != 0
```

Comment:
Won't make a big difference, but why not.
```suggestion
	return b < utf8.RuneSelf && regexMetaCharacterBytes[b%16]&(1<<(b>>4)) != 0
```

Reply:
This code came from the Go source, probably best not to change it.